src/main/java/CarRacing/Racing.java

```diff
@@ -0,0 +1,65 @@
+package CarRacing;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Racing {
+
+    private final int MAX_NAME_LENGTH = 5;
+    public String[] nameSplit(String inputString) {
+        return inputString.split(",");
+    }
+
+    public void nameException(String name) {
+        if (name.length() > MAX_NAME_LENGTH) {
+            throw new RuntimeException("다섯 글자 이상입니다.");
+        }
+    }
+
+    public void nameExceptions(String[] names) {
+        for (int index = 0; index < names.length; index++) {
+            nameException(names[index]);
+        }
+    }
+
+    public Car[] makeCarList(String[] names) {
+        Car[] cars = new Car[names.length];
+        for (int index = 0; index < cars.length; index++) {
+            cars[index] = new CarBuilder().name(names[index]).buildCar();
+        }
+        return cars;
+    }
+
+    public List<String> getWinner(Car[] cars) {
+        List<String> winners = new ArrayList<>();
+        int maxPosition = 0;
+        for (int index = 0; index < cars.length; index++) {
+            maxPosition = cars[index].getMaxPosition(maxPosition);
+        }
```

Comment:
getMaxPosition 네이밍과 구현이 헷갈릴 수 있을 것 같은 부분이 있네요!
stream을 사용하면 아래와 같이도 구현할 수 있을 것 같아요! 😄 
이렇게 개선되면 getMaxPosition 메서드 없이도 구현이 가능하지 않을까요? :)

```suggestion
        int maxPosition = Arrays.stream(cars)
                .mapToInt(Car::getPosition)
                .max()
                .orElse(0);
```